src/screens/OtherProfile/index.js

```diff
@@ -255,8 +256,8 @@ const OtherProfile = () => {
       if (dataMain.is_following)
         return (
           <GlobalButton onPress={() => handleSetUnFollow()}>
-            <View style={styles.buttonFollowing}>
-              <Text style={styles.textButtonFollowing}>Following</Text>
+            <View style={styles.buttonFollowing(dynamicColor)}>
+              <Text style={styles.textButtonFollowing(dynamicColor)}>Following</Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `buttonFollowing` and `textButtonFollowing` styles now accept `dynamicColor` as an argument. Ensure that `dynamicColor` is correctly defined and passed where these styles are used.
```diff
- <View style={styles.buttonFollowing}>
-   <Text style={styles.textButtonFollowing}>Following</Text>
+ <View style={styles.buttonFollowing(dynamicColor)}>
+   <Text style={styles.textButtonFollowing(dynamicColor)}>Following</Text>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->